### PR TITLE
Resets

### PR DESCRIPTION
**Pull request by g33kex**
This puts back the player to title screen when
- Isaac wins
- Isaac dies

we'll do later multiple screens and store Isaac state later when #136 is fully fixed. 

It also adds alternance between Isaac eyes, allowing more than 10 Isaac tears on the same line (because they now are shifted by at least 4 pixels)

### CODE REVIEW

**Comment by g33kex** [comment]
Also, goodnight!

File: src/check_collisions_isaac_enemies.s [hunk 1]
```diff
@@ -74,11 +74,13 @@ enemies_collide:
 	ld a, (global_.isaac.hp)
 	sub b
 	ld (global_.isaac.hp), a
-	bit 7, a
-	jr z, @noDeath
```
Comment (histausse) [inline src/check_collisions_isaac_enemies.s:78]: That looks wrong. If your hp goes to negative you don't die here. You should check both zero **and** sign.

**Review by histausse — CHANGES_REQUESTED** [review]
Check the hp for negative value as well. 
And please, fix #130

File: src/check_collisions_isaac_enemies.s (hunk 1 above)
Comment (g33kex) [reply src/check_collisions_isaac_enemies.s:78]: Fixed.

**Review by histausse — APPROVED** [review]

**Pull request merged**